hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/EncryptionZoneManager.java

```diff
@@ -645,8 +645,8 @@ private boolean pathResolvesToId(final long zoneId, final String zonePath)
   /**
    * Re-encrypts the given encryption zone path. If the given path is not the
    * root of an encryption zone, an exception is thrown.
-   * @param zoneIIP
-   * @param keyVersionName
+   * @param zoneIIP encryption zone inode path
```

Comment:
nit: `encryption zone inodes in the path containing the file`

Reply:
Thanks @virajjasani  for the comment.
I will update it later.